components/omega/doc/design/OmegaV0ShallowWater.md

```diff
@@ -0,0 +1,537 @@
+
+### 2.9  Performance will be at least as good or better than MPAS-Ocean
+
+Performance will be assessed with the following metrics:
+
+1. Single CPU throughput
+1. Parallel CPU scalability to high node counts
+1. Single GPU throughput
+
+The first two items will be compared to MPAS-Ocean with the same test configuration. For Omega-0, these may be tested first with the layered shallow water equations (momentum and thickness only) and be compared directly to the results in [Bishnu et al. 2023](https://gmd.copernicus.org/articles/16/5539/2023), using the inertia-gravity wave test case described in Section 6.2 below. After tracer transport capabilities are added, performance may be compared against MPAS-Ocean with test cases using active tracers in addition to momentum and thickness.
+
+For these comparisons, MPAS-Ocean will be set up with the identical terms and functionality as Omega-0. This means that vertical advection, vertical mixing, and all parameterizations will be disabled. Comparisons with these terms will be made with Omega-1 and higher and will be described in future design documents.
+
+### 2.10  Full-node GPU throughput will be comparable or better than full-node CPU throughput
+
+For GPU throughput, comparisons should be made between full-node CPU throughput and full-node GPU throughput. For example, [Perlmutter at NERSC](https://docs.nersc.gov/systems/perlmutter/architecture/) has nodes with 64 and 128 CPU cores (AMD EPYC 7763) and 4 GPUs (NVIDIA A100). We expect the full-node GPU throughput to be at least as good as the full-node CPU throughput, and potentially a factor of four higher. These numbers depend on the performance specifications of the particular hardware.
+
+Like the previous requirement, tests will first be conducted with layered shallow water equations and later with additional tracer advection. For reference, [Bishnu et al. 2023](https://gmd.copernicus.org/articles/16/5539/2023) was able to obtain nearly identical throughput between 64 CPU cores and a single GPU on Perlmutter using a Julia code and a layered shallow water test case.
+
+## 3 Algorithmic Formulation
+
+
+### 3.1 Governing Equations
+
+#### 3.1.1 Continuous Equations
+
+The algorithms for Omega-0 are given here in full detail. The original version with derivations are given in [Thuburn et al. 2009](https://www.sciencedirect.com/science/article/pii/S0021999109004434) and [Ringler et al. 2010](https://www.sciencedirect.com/science/article/pii/S0021999109006780).
+In continuous form, these are
+
+$$
+\frac{\partial \boldsymbol{u}}{\partial t} + \left(\boldsymbol{u}\cdot\nabla\right)\boldsymbol{u}  + f \boldsymbol{k}\times\boldsymbol{u}
+= -g\nabla(h+b)
```

Comment:
Shouldn't `(h+b)` be `(h+H)` according to the definitions of `H` in the table below? Or need to change `H` in the table to `b` as bottom topography?

Reply:
Done. I changed H to b.